packages/components/src/tabs/index.tsx

```diff
@@ -163,9 +189,11 @@ function Tabs( {
 	);
 
 	return (
-		<TabsContext.Provider value={ contextValue }>
-			{ children }
-		</TabsContext.Provider>
+		<div ref={ tabsRef }>
```

Comment:
You could check `item.element`:

```js
const tabsHasFocus = items.some( ( item ) => {
  const activeElement = item.element?.ownerDocument.activeElement;
  return activeElement && activeElement === item.element;
} );
```

However, I think that should be done inside `useEffect` rather than on render.

Reply:
Thank you both! Using `item.element` was a great idea. I had to tweak the approach slightly, because I needed to rescope `activeElement` so I could access it again in a different scope, but the changes in ea1e3233e0 have this working without adding a `div` solely to carry a ref around.